index.html

```diff
@@ -436,13 +436,20 @@ <h3>Use Cases and Requirements</h3>
 <!-- requirement list start -->
         <ul>
           <li>
+          <li>
+<a>Credentials</a> represent statements made by an issuer.
+          </li>
 <a>Verifiable credentials</a> represent statements made by an <a>issuer</a> in
 a tamper-evident and privacy-respecting manner.
           </li>
           <li>
 <a>Holders</a> assemble collections of <a>verifiable credentials</a> from
 different <a>issuers</a> into a single artifact, a
-<a>verifiable presentation</a>.
+<a>presentation</a>.
+          </li>
+          <li>
+Holders convert <a>presentations</a> into <a>verifiable presentations</a>,
+rendering them tamper-evident. 
```

Comment:
Slight tweak on @msporny's suggestion that might be simpler language:

```suggestion
Holders transform <a>presentations</a> into <a>verifiable presentations</a>
to render them tamper-evident. 
```

Reply:
[This](https://github.com/w3c/vc-data-model/pull/808/files#r704704940) does not appear to have been applied.

Reply:
I have tried to commit this but it fails to do so.